src/node.js

```diff
@@ -43,6 +43,9 @@
 
     startup.processRawDebug();
 
+    // Load async_wrap to init the wrapper class info provider.
+    process.binding('async_wrap');
```

Comment:
I'd prefer to just run `SetWrapperClassInfoProvider()` from `node.cc`, but then I'd have to migrate all the `RetainedAsyncInfo` logic as well. Thoughts on a better solution?